metadata-ingestion/src/datahub/ingestion/source/state/ldap_state.py

```diff
@@ -0,0 +1,50 @@
+from typing import Iterable, List
+
+import pydantic
+
+from datahub.ingestion.source.state.stale_entity_removal_handler import (
+    StaleEntityCheckpointStateBase,
+)
+
+
+class BaseLdapCheckpointState(
```

Comment:
Could you rename this to just `LdapCheckpointState` without the `Base`?

Reply:
Sure, will do

Reply:
completed